BaseFilter/sections/antiadblock.txt

```diff
@@ -1072,7 +1072,14 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 !
 ! NOTE: Regular rules
 !
-!
+! https://github.com/AdguardTeam/AdguardFilters/issues/176432
+biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
+!+ PLATFORM(ios, ext_android_cb)
+@@||biolink.webtena.com^$generichide
+!https://github.com/AdguardTeam/AdguardFilters/issues/176383

```

Comment:
```suggestion
! https://github.com/AdguardTeam/AdguardFilters/issues/176383

```
Please add a space after `!` in comments (not ditectives or hints)

Reply:
I've added the space